:farmer: Error C2065: 'M_PI': undeclared identifier in AdvancedLiftDrag.cc(618,64)

## Environment
* OS Version: Windows CI
* Source gz-sim build

## Description
* Expected behavior: Build should pass
* Actual behavior: Build is failing because of an undeclared identifier

## Steps to reproduce
1. Run a job in gz-sim7 or 8
2. See build fail

## Output
Reference build: [gz-sim8-win](https://build.osrfoundation.org/job/ign_gazebo-gz-8-win/28/consoleFull#12751072418ff58640-3599-4406-a210-216932f1748c)

Log output:
```
  gz-sim8-acoustic-comms-system.vcxproj -> C:\J\workspace\ign_gazebo-gz-8-win\ws\build\gz-sim8\bin\Release\gz-sim8-acoustic-comms-system.dll

  Building Custom Rule C:/J/workspace/ign_gazebo-gz-8-win/ws/gz-sim/src/systems/advanced_lift_drag/CMakeLists.txt

  AdvancedLiftDrag.cc

C:\J\workspace\ign_gazebo-gz-8-win\ws\gz-sim\src\systems\advanced_lift_drag\AdvancedLiftDrag.cc(618,64): error C2065: 'M_PI': undeclared identifier [C:\J\workspace\ign_gazebo-gz-8-win\ws\build\gz-sim8\src\systems\advanced_lift_drag\gz-sim8-advanced-lift-drag-system.vcxproj]

C:\J\workspace\ign_gazebo-gz-8-win\ws\gz-sim\src\systems\advanced_lift_drag\AdvancedLiftDrag.cc(674,46): error C2065: 'M_PI': undeclared identifier [C:\J\workspace\ign_gazebo-gz-8-win\ws\build\gz-sim8\src\systems\advanced_lift_drag\gz-sim8-advanced-lift-drag-system.vcxproj]
```

`M_PI` mentions: https://github.com/gazebosim/gz-sim/blob/gz-sim8/src/systems/advanced_lift_drag/AdvancedLiftDrag.cc#L618 and https://github.com/gazebosim/gz-sim/blob/gz-sim8/src/systems/advanced_lift_drag/AdvancedLiftDrag.cc#L674.

This is only failing in Windows

--- a/examples/standalone/marker/marker.cc
+++ b/examples/standalone/marker/marker.cc
@@ -172,7 +172,7 @@ int main(int _argc, char **_argv)
   gz::msgs::Set(markerMsg.add_point(),
         gz::math::Vector3d(0, 0, 0.05));
   double radius = 2;
-  for (double t = 0; t <= M_PI; t+= 0.01)
+  for (double t = 0; t <= GZ_PI; t+= 0.01)
   {
     gz::msgs::Set(markerMsg.add_point(),
         gz::math::Vector3d(radius * cos(t), radius * sin(t), 0.05));

--- a/src/systems/advanced_lift_drag/AdvancedLiftDrag.cc
+++ b/src/systems/advanced_lift_drag/AdvancedLiftDrag.cc
@@ -615,7 +615,7 @@ void AdvancedLiftDragPrivate::Update(EntityComponentManager &_ecm)
     {
       components::JointPosition *tmp_controlJointPosition =
       controlJointPosition_vec[i];
-      controlAngle = tmp_controlJointPosition->Data()[0] * 180/M_PI;
+      controlAngle = tmp_controlJointPosition->Data()[0] * 180 / GZ_PI;
     }
 
     // AVL's and Gazebo's direction of "positive" deflection may be different.
@@ -671,7 +671,7 @@ void AdvancedLiftDragPrivate::Update(EntityComponentManager &_ecm)
 
   double CD_fp = 2 / (1 + exp(this->CD_fp_k1 + this->CD_fp_k2 * (
     std::max(this->AR, 1 / this->AR))));
-  CD = (1 - sigma) * (this->CD0 + (CL*CL) / (M_PI * this->AR *
+  CD = (1 - sigma) * (this->CD0 + (CL*CL) / (GZ_PI * this->AR *
     this->eff)) + sigma * abs(
       CD_fp * (0.5 - 0.5 * cos(2 * this->alpha)));
 

--- a/test/integration/added_mass.cc
+++ b/test/integration/added_mass.cc
@@ -50,13 +50,13 @@ const double kRate = 1000;
 const double kForceVec[3] = {2000, 2000, 0};
 
 // Force excitation angular velocity [rad / s].
-const double kForceAngVel = 3 * M_PI;
+const double kForceAngVel = 3 * GZ_PI;
 
 // Torque excitation amplitude and direction.
 const double kTorqueVec[3] = {200, 200, 0};
 
 // Torque excitation angular velocity [rad / s].
-const double kTorqueAngVel = 2 * M_PI;
+const double kTorqueAngVel = 2 * GZ_PI;
 
 // Total duration of the motion in iterations.
 const uint64_t kIter = 1000;

